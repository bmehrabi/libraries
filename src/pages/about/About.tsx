import React from 'react';
import Container from "@mui/material/Container";
import Grid from "@mui/material/Grid";
import Paper from "@mui/material/Paper";

export const About = () => {
  return (
    <Container maxWidth="lg" sx={{ mt: 4, mb: 4 }}>
      <Grid container>
        {/* About */}
        <Grid item xs={12}>
          <Paper
            sx={{
              p: 2,
              display: 'flex',
              flexDirection: 'column',
              height: 240,
            }}
          >
            <h2>About Me</h2>

            <p>
              My name is Babak Mehrabi and I did this task as an interview task for Wire.
            </p>
            <p>
              If you have any question or comment regarding this project, you can write me an email:
              <br/>
              <b>b.mehrabi.91 [at] gmail.com</b>
            </p>
          </Paper>
        </Grid>
      </Grid>
    </Container>
  );
}