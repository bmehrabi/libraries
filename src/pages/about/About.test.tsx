import React from 'react';
import {render} from '@testing-library/react';
import {About} from "./About";

test('matches the snapshot', () => {
    const {container} = render(<About />);

    expect(container).toMatchSnapshot();
});
