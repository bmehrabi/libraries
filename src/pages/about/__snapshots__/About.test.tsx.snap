// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1f2kw6w-MuiContainer-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-t5ouwv-MuiPaper-root"
        >
          <h2>
            About Me
          </h2>
          <p>
            My name is Babak Mehrabi and I did this task as an interview task for Wire.
          </p>
          <p>
            If you have any question or comment regarding this project, you can write me an email:
            <br />
            <b>
              b.mehrabi.91 [at] gmail.com
            </b>
          </p>
        </div>
      </div>
    </div>
  </div>
</div>
`;
